drupal/rootfs/etc/cont-init.d/99-custom-setup.sh

```diff
@@ -163,7 +162,7 @@ function main {
   if [ -n "${DRUPAL_INSTANCE}" ] && [ "${DRUPAL_INSTANCE}" != "dev" ] ;
   then
     # TODO:  This fails and needs to be fixed or factored out
-    perform_runtime_config "${site_url}" || printf "\n\nWARNING: runtime config failed, ignoring\n\n"
+    perform_runtime_config "${site}" || printf "\n\nWARNING: runtime config failed, ignoring\n\n"
```

Comment:
Great - can you remove the `|| printf ...` part?  Now that it is fixed, we'd want this to fail if `perform_runtime_config` fails.  Prior to this fix, the `|| printf` was in place as a temporary workaround to allow `perform_runtime_config` to fail, without terminating the script

Reply:
yup! good catch.